### ISSUE TITLE
SettingsFlyout positioned incorrectly while transitioning in.

### ISSUE THREAD

**Issue by TheRoboZ** [timheuer/callisto#119]
if i open a SettingsFlyout without invoking it form the charms, the position during the initial transition animation is wrong for some frames (it starts on the left side of the screen). See the screenshot here:

http://www.bigbangpixel.com/Forum/CallistoBug.png

Then, when it;s fully transitioned, it is correctly anchored to the right side of the screen.

If i open it from the charm the animation is correct, on the right side. Should I set something that is set automatically when invoked from the charm? 

I am using version 1.2.5 but this happened also with 1.2.4.

My code is the following and it;s the same form charm or not:

``` c#
SettingsFlyout settingsFlyout = new SettingsFlyout();

settingsFlyout.HeaderBrush = new  Windows.UI.Xaml.Media.SolidColorBrush(TILE_COLOR);

settingsFlyout.HeaderText = SkiddyEngine.SkiddyStrings.GetString("Skiddy");

settingsFlyout.SmallLogoImageSource =new BitmapImage(new Uri(STORE_LOGO));

SettingsContent settings = new SettingsContent(); //Just 2 togggle switches

settingsFlyout.Content = settings;

settingsFlyout.IsOpen = true;
```

Thanks and good job on Callisto!

**Comment by timheuer** [comment]
I just tried the above from a button on my page and cannot repro the issue.  Do you have other animations in your SettingsContent user control that might be firing?

**Comment by TheRoboZ** [comment]
Hi Tim, I was also trying to reproduce it with Callisto's test app simulating the 2 situations (from charm and invoked directly) as in our app and i noticed it doesn't happen there. We don't have any additional animation in the SettingsContent, it just contains 2 toggle switches and labels but our game is a conversion from an XNA windows phone game using monogame to run on windows 8 as a metro app, maybe there is something going on with monogame that is conflicting with the SettingsFlyout in this special situation. I just got Callisto's repository from github and do some additional tests then let you know if I found anything useful.

Thanks,

Andrea
